jbake/content/what-is-shiro.md

```diff
@@ -1,10 +1,13 @@
-<a name="WhatisShiro-WhatisShiro%3F"></a>
-#What is Shiro?
+title=What is Shiro?
+type=page
+tags=documentation, about
+status=published
+~~~~~~
 
 Apache Shiro is an application security framework that provides application developers very clean and simple ways of supporting four cornerstones of security in their applications: authentication, authorization, enterprise session management and cryptography.
 
 <a name="WhatisShiro-MissionStatement"></a>
```

Comment:
Should we drop all of these named anchors and just use the generated ones? (unless there is a specific cross-page reference?)

Reply:
> Should we drop all of these named anchors and just use the generated ones? (unless there is a specific cross-page reference?)

I'd say yes, *even if* there is a cross reference. I will check all links anyway after all pages have been moved over via https://github.com/filiph/linkcheck.

Good URLs never change, but anchors… they can change.

But that is just my opinion. Feel free to keep the anchors! https://docs.asciidoctor.org/asciidoc/latest/attributes/ids/